sdk/batch/Azure.ResourceManager.Batch/tests/TestCase/BatchAccountCertificateCollectionTests.cs

```diff
@@ -8,32 +8,55 @@
 using Azure.Core.TestFramework;
 using Azure.ResourceManager.Batch.Models;
 using Azure.ResourceManager.Batch.Tests.Helpers;
+using Azure.ResourceManager.Resources;
 using NUnit.Framework;
 
 namespace Azure.ResourceManager.Batch.Tests.TestCase
 {
     public class BatchAccountCertificateCollectionTests : BatchManagementTestBase
     {
+        private ResourceIdentifier _batchAccountId;
+        private BatchAccountResource _batchAccountResource;
+
         public BatchAccountCertificateCollectionTests(bool isAsync)
             : base(isAsync)//, RecordedTestMode.Record)
         {
         }
 
-        private async Task<BatchAccountCertificateCollection> GetAccountCollectionAsync()
+        [OneTimeSetUp]
+        public async Task GlobalSetup()
+        {
+            var rgName = SessionRecording.GenerateAssetName("testrg-batch");
+            var storageAccountName = SessionRecording.GenerateAssetName("azstorageforbatch");
+            var batchAccountName = SessionRecording.GenerateAssetName("testaccount");
+            if (Mode == RecordedTestMode.Playback)
+            {
+                _batchAccountId = BatchAccountResource.CreateResourceIdentifier(SessionRecording.GetVariable("SUBSCRIPTION_ID", null), rgName, batchAccountName);
+            }
+            else
+            {
+                using (SessionRecording.DisableRecording())
+                {
+                    var rgLro = await (await GlobalClient.GetDefaultSubscriptionAsync()).GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Started, rgName, new ResourceGroupData(AzureLocation.WestUS2));
+                    var storage = await CreateStorageAccount(rgLro.Value, storageAccountName);
+                    var batchAccount = await CreateBatchAccount(rgLro.Value, batchAccountName, storage.Id);
+                    _batchAccountId = batchAccount.Id;
```

Comment:
Should we include the creation of this resource in our recording because it is BATCHAccount which is a resource we should test in this RP?

Reply:
At first, considering that the recording should focus on the resource that we are testing, I didn't include this part in the recording. But it also makes sense to include it, updated.